packages/core/src/index.ts

```diff
@@ -88,6 +89,8 @@ import {
   addImageSlicesToViewports,
 } from './RenderingEngine/helpers';
 
+export * from './loaders/enhancedVolumeLoader';
```

Comment:
i'm not sure if enhancedVolumeLoader is a good name, maybe decimatedVolumeLoader?